pages/measurement-sets/[id].js

```diff
@@ -279,18 +278,18 @@ export default function MeasurementSet({
                       </DataItemValue>
                     </>
                   )}
-                  {measurementSet.protocol && (
+                  {measurementSet.protocols.length > 0 && (
                     <>
-                      <DataItemLabel>Protocol</DataItemLabel>
-                      <DataItemValueUrl>
-                        <a
-                          href={measurementSet.protocol}
-                          target="_blank"
-                          rel="noopener noreferrer"
-                        >
-                          {measurementSet.protocol}
-                        </a>
-                      </DataItemValueUrl>
+                      <DataItemLabel>Protocol(s)</DataItemLabel>
```

Comment:
I’d like to avoid using the “(s)” suffix in IGVF and instead either always use plural even if only one exists, or to make the pluralization conditional based on the count. I think always using “Protocols” here works even if only one exists. The conditional case works better if you have something like “You have {x} protocol.”

Reply:
done